charts/agent-operator/templates/operator-clusterrole.yaml

```diff
@@ -14,12 +14,25 @@ rules:
   - logsinstances
   - podlogs
   verbs: [get, list, watch]
+- apiGroups: [monitoring.grafana.com]
+  resources:
+  - grafanaagents/finalizers
+  - metricsinstances/finalizers
+  - logsinstances/finalizers
+  - podlogs/finalizers
+  verbs: [get, list, watch, create, update, patch, delete]
```

Comment:
```suggestion
  verbs: [get, list, watch, update]
```

Reply:
Still seems to work with reduced scope, just pushed these changes.